components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -452,6 +452,9 @@ - (void)setItems:(NSArray<UITabBarItem *> *)items {
     [itemView.button addTarget:self
                         action:@selector(didTouchUpOutsideButton:)
               forControlEvents:UIControlEventTouchUpOutside];
+    [itemView.button addTarget:self
+                        action:@selector(didTouchUpOutsideButton:)
+              forControlEvents:UIControlEventTouchCancel];
```

Comment:
You don't want animation for cancelations.

Reply:
Why not? It looks good for the use cases I could think of:

* Tap-and-hold a tab item and then rotate the device (ink dissipates)
* Tap-and-hold a tab item, then swipe down and swipe up to show the Control Panel.

Is there a use case where we are canceling touch events and Ink should not dissipate?

Reply:
Switched to immediately cancel ink after further discussion with @randallli and @yarneo .

Reply:
This should work the same way we cancel the ink when a collection view scroll gesture may initially start ink.